src/hotspot/cpu/s390/nativeInst_s390.cpp

```diff
@@ -385,7 +385,7 @@ intptr_t NativeMovConstReg::data() const {
 #else
     ShouldNotReachHere();
 #endif
-    return *(intptr_t *)NULL;
+    return *(intptr_t *)nullptr;
```

Comment:
Unnecessary cast? Also, this is de-referencing a null pointer, so we might have to do this differently after the conversion.